ports/stm/boards/swan_r5/mpconfigboard.mk

```diff
@@ -71,5 +70,8 @@ CIRCUITPY_BUSDEVICE = 0
 CIRCUITPY_KEYPAD = 1
 CIRCUITPY_RGBMATRIX = 0
 CIRCUITPY_RTC = 1
-
 CIRCUITPY_BUILD_EXTENSIONS = bin,uf2
+CIRCUITPY_ENABLE_MPY_NATIVE = 1
+CIRCUITPY_AUDIOBUSIO = 1
+CIRCUITPY_AUDIOBUSIO_I2SOUT = 0
+CIRCUITPY_AUDIOBUSIO_PDMIN = 1
```

Comment:
Move these up into lines 41-46, with the other enabling/disabling values, and consider alphabetizing those lines.

Reply:
This is done - all module names in alphabetical order.